default/python/universe_generation/universe_generator.py

```diff
@@ -144,6 +144,19 @@ def create_universe(psd_map):
     seed_rng(seed_pool.pop())
     distribute_specials(gsd.specials_frequency, fo.get_all_objects())
 
+    # set game uid
+    empire_names = []
```

Comment:
You can do all operations in single line:
`empire_names  = sorted(x.capitalize()[:2] for x in psd_map.values())`

Reply:
I'm not Python coder.

Reply:
You write python code, so you are.  :) 

Nobody expects a single person to have all the skills,  but as the team, we have all the skills :)